Makefile

```diff
@@ -24,17 +24,25 @@ deps:
 checks:
 	./scripts/checks "$(PKGS)"
 
-build: deps checks
```

Comment:
@erikh IIRC from out last discussion we wanted to do host native build inside a container and then have a build iniside vm (just for mac users).

Just wondering if you want to address that as part of this PR? as otherwise the build might fail for folks trying out netplugin inside a vm.